src/enc.cpp

```diff
@@ -87,12 +90,22 @@ int emain()
     }
```

Comment:
```c
    if (is_marblerun &&
        mount("/", "/edg/hostfs", OE_HOST_FILE_SYSTEM, 0, nullptr) != 0)
```
that's some lines above where I can't comment

Reply:
As discussed, cannot apply this change yet since we cannot set a `memfs` path as cwd. Need to address this in an upcoming PR before release.